sdk/python/t4/api.py

```diff
@@ -374,7 +374,7 @@ def config(*autoconfig_url, **config_values):
     """Set or read the Helium configuration
 
     To retrieve the current config, call directly, without arguments:
-        >>> import helium as he
+        >>> import t4 as he
         >>> he.config()
```

Comment:
We should probably call it `t4` rather than `he` now?

Reply:
Yeah, going to do a follow up rename of all these and the Exceptions names and docs.